lib/index.js

```diff
@@ -1,3 +1,37 @@
-const bindings = require('bindings');
+/**
+ * Equihash for Node.js.
+ * Main entry point.
+ *
+ * Copyright (c) 2017 Digital Bazaar, Inc.
+ *
+ * MIT License
+ * <https://github.com/digitalbazaar/equihash/blob/master/LICENSE>
+ */
+function engine(name=engine.default) {
+  return require('./' + name);
+}
+engine.default = 'khovratovich';
 
-module.exports = engine => require('./' + engine);
+module.exports = {
+  engine,
+  solve() {
+    return engine().solve(...arguments);
+  },
+  verify() {
+    return engine().verify(...arguments);
+  },
+  get PERSONALBYTES() {
+    return engine().PERSONALBYTES;
+  },
+  zcashPersonal(n, k) {
```

Comment:
Maybe this should be `_zcashPersonal` as a private helper thing we're not exposing as part of the public API right now until we figure it out better. I would instead expect to find this, in the future, under `.engine('zcash').PERSONAL_BYTES` (we aren't implementing it now).